### PR TITLE
feat: Add `appTimeZone` capability

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
@KazuCocoa could you please verify if the cap also works on real devices or it's only limited to simulators?

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Sure, will check it

File: docs/reference/capabilities.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ about capabilities, refer to the [Appium documentation](https://appium.io/docs/e
...
+| `appium:appTimeZone` | Defines the custom time zone override for the application under test. You can use UTC, PST, EST, as well as place-based timezone names such as America/Los_Angeles. The application must be (re)launched for the capability to take effect. See the [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) for more details. The same behavior could be achieved by providing a custom value to the [TZ](https://developer.apple.com/forums/thread/86951#263395) environment variable | UTC |
```
Comment (KazuCocoa) [inline docs/reference/capabilities.md:40]:
```suggestion
| `appium:appTimeZone` | Defines the custom time zone override for the application under test. You can use UTC, PST, EST, as well as place-based timezone names such as America/Los_Angeles. The application must be (re)launched for the capability to take effect. See the [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) for more details. The same behavior could be achieved by providing a custom value to the [TZ](https://developer.apple.com/forums/thread/86951#263395) environment variable with `appium:processArguments` | UTC |
```

Adding a guidance for env var would be helpful?

**Review by KazuCocoa — APPROVED** [review]
it worked on a real device as well

**Comment by KazuCocoa** [comment]
tested with `com.apple.mobilecal` and `EST`, for example. Just a note.

File: docs/reference/capabilities.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/reference/capabilities.md]: added

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**